pluto_build/bin/setup.sh

```diff
@@ -4,17 +4,8 @@ function setup {
 sudo apt update
 sudo apt install -y curl zip
 
-sudo tee /etc/apt/sources.list.d/pgdg.list << END
-deb http://apt.postgresql.org/pub/repos/apt/ bionic-pgdg main
-END
-# get the signing key and import it
-curl -O https://www.postgresql.org/media/keys/ACCC4CF8.asc
-sudo apt-key add ACCC4CF8.asc
-
-sudo apt update
-sudo apt install -y postgresql-client-11 gdal-bin
-sudo apt autoremove -y
-rm ACCC4CF8.asc
+# install pip for the workflow to run correctly
+sudo apt install python3-pip
```

Comment:
I had no idea this setup.sh file existed lol. I can't figure out where the `setup` bash function is run. Is it in the last line with the `register` call?

Reply:
see my answer below for this.